src/frontend/Player/createPlayer.ts

```diff
@@ -12,11 +12,12 @@ export const createPlayer: VideoPlayerCreator = (
   switch (type) {
     case 'plyr':
       return createPlyrPlayer(ref, dispatchPlayerTimeUpdate, video);
-    case 'videojs':
+    case 'videojs': {
```

Comment:
IMO this is not necessarily a good change as it seems to prevent stacking `case` instructions, which is IMO one of the best features of `switch`